questions/qa-resizing-backgrounds.en.html

```diff
@@ -47,31 +47,30 @@
 <header>
   <nav id="mainNavigation"></nav><script>document.getElementById('mainNavigation').innerHTML = mainNavigation</script>
   
-  <h1>Background images that support localization</h1>
+  <h1>Historial approaches to rounded corners</h1>
 </header>
 
 
   <div id="audience">
     <div id="updateInfo"></div><script>document.getElementById('updateInfo').innerHTML = g.updated</script>
     </div>
-    
-<section id="question">
-<h2>Question</h2>
-    <p class="question">How can I ensure that when text expands in translation the background images will still work?</p>
-</section>
 
+    <div class="note">
+      <p><strong>NOTE!</strong> This is a historical article, which describes the best practice before the existence of the <a href="https://www.w3.org/TR/css-backgrounds-3/#corners">border-radius</a> properties.
```

Comment:
Suggestions:
- drop the word 'the' between 'describes' and 'best'
- properties -> property

Reply:
> drop the word 'the' between 'describes' and 'best'

Fixed.

> properties -> property

I copied the word from [CSS](https://www.w3.org/TR/css-backgrounds-3/#corners) because there is more than one property.